File: Exercise/Easy/12. Calculate Perimeter of Rectangle/Code/Calculate Perimeter of Rectangle.py
import sys

def calculate_perimeter(length, width):
    result = 2 * (length + width)
    print(result)

if __name__ == "__main__":
    length, width = map(int, sys.stdin.readline().split())

    calculate_perimeter(length, width)

    sys.exit(0)

File: Exercise/Easy/12. Calculate Perimeter of Rectangle/Code/Calculate Perimeter of Rectangle.cs
using System;

public class CalcPer {
    public static void calcPer(int length, int width) {
        int result = 2 * (length + width);
        Console.WriteLine(result);
    }

    public static void Main(string[] args) {
        int length = int.Parse(Console.ReadLine());
        int width = int.Parse(Console.ReadLine());

        calcPer(length, width);
    }
}

